### DISCUSSION TITLE
No route found for "GET

### DISCUSSION THREAD

**Discussion by ghost** [uvdesk/community-skeleton#718] [General]
Good morning,
I'm trying to open an image/document from a ticket that was generated in uvdesk but when opening this error message appears, I can't see the image/document i have to go to this folder

public\assets\threads

![image](https://github.com/uvdesk/community-skeleton/assets/158157781/0bcc48f6-bbf7-451c-a362-48f0dd42643d)

**Comment by PeopleInside** [comment]
Hi, the UVdesk support seems away in those weeks and I don't know when they will back and if they will.
Maybe you are not able to download the attachment because of the bug reported here: https://github.com/uvdesk/community-skeleton/issues/710 ?

**Comment by ghost** [comment]
Thank you very much, this fix helped me

**Reply by PeopleInside** [reply]
What fix?
The topic I mentioned is an open issue, I don't see any fix for this issue currently.

Oh maybe what was mentioned in the forum. I never tried :)
I'm glad to read you was able to resolve.
